aiida_vasp/calcs/base.py

```diff
@@ -59,23 +59,23 @@ def max_retrieve_list(cls):
             'wannier90*',  # Wannier90 related
             'OSZICAR',  # Convergence related
             'REPORT',  # Output of molecular dynamics runs
-            'STOPCAR',  # Controlled stopping file
+            'STOPCAR',  # Controlled stopping
             'vasprun.xml',
             'OUTCAR'
         ]
         return retrieve_list
 
     def prepare_for_submission(self, tempfolder):  # pylint: disable=arguments-differ
         """
-        Writes the four minimum output files, INCAR, POSCAR, POTCAR, KPOINTS.
+        Writes the four minimum outputs: INCAR, POSCAR, POTCAR, KPOINTS.
 
-        Delegates the construction and writing / copying to write_<file> methods.
+        Delegates the construction and writing / copying to write_<object> methods.
         That way, subclasses can use any form of input nodes and just
         have to implement the write_xxx method accordingly.
         Subclasses can extend by calling the super method and if necessary
         modifying it's output CalcInfo before returning it.
         """
-        # write input files
+        # write input objects
```

Comment:
'fiies' rather than 'objects'?

Reply:
Ye, what I was thinking was in fact for these cases to remove it altogether and just write `write input`. If input change, e.g. if there will ever be an API to VASP it can still be similar.